examples/elm-package/README.md

```diff
@@ -0,0 +1,238 @@
+# What
```

Comment:
Consider moving this file to `/tutorials/elm-package/README.md` as the `/examples` is more for basic code examples whereas `/tutorials` appears to be for more long-form step-by-step walk through.